vividus-tests/src/main/resources/properties/environment/system/saucelabs/mobile_app/ios/environment.properties

```diff
@@ -3,6 +3,8 @@ selenium.grid.platform-version=13.4
 
 bdd.variables.global.target-platform=ios
 
+bdd.variables.global.element-to-ignore=xpath(//XCUIElementTypeOther[./XCUIElementTypeOther/XCUIElementTypeImage])
```

Comment:
lets assign the image id? :)

Reply:
To use an image as a first step it would be nice to understand why it's x is -13 :)